docs/migration-prep/vultr_startup.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Vultr Startup Script for SGLang-Mamba A100 VM
+
+# 1. System Updates & Dependencies
+export DEBIAN_FRONTEND=noninteractive
+apt-get update
+apt-get install -y git python3-pip python3-venv curl wget
+
+# 2. Setup Repository
+cd /root
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)